uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/test/java/org/uberfire/java/nio/fs/jgit/daemon/ssh/GitSSHServiceTest.java

```diff
@@ -234,7 +252,9 @@ public void testCheckSetupParameters() throws Exception {
                              "RSA",
                              null,
                              mock(JGitFileSystemProvider.RepositoryResolverImpl.class),
-                             executorService);
+                             executorService,
+                             null,
```

Comment:
not liking all these nulls here. if preserve the original tests without params, I'd prefer to add an overload constructor

Reply:
Ok

Reply:
Ok